### PR TITLE
generate book sitemap.

### PR DESCRIPTION
**Pull request by jetfuel**

### CODE REVIEW

**Comment by jetfuel** [comment]
https://github.com/PaddlePaddle/PaddlePaddle.org/issues/213

File: portal/deploy/sitemap_generator.py [hunk 1]
```diff
@@ -53,3 +55,47 @@ def models_sitemap(original_documentation_dir, generated_documentation_dir, vers
 
 def _get_destination_documentation_dir(version, output_dir_name):
     return '%s/docs/%s/%s' % (settings.EXTERNAL_TEMPLATE_DIR, version, output_dir_name)
+
+
+def _book_sitemap_with_lang(original_documentation_dir, generated_documentation_dir, version, output_dir_name, lang):
+    title = 'Deep Learning 101'
+    root_json_path_template = '.tools/templates/index.html.json'
+    output_file_name = 'site.json'
```
Comment (cs2be) [inline portal/deploy/sitemap_generator.py]: site.en.json

File: portal/deploy/sitemap_generator.py [hunk 2]
```diff
@@ -53,3 +55,47 @@ def models_sitemap(original_documentation_dir, generated_documentation_dir, vers
...
+
+    if lang == 'zh':
+        title = '深度学习入门'
+        root_json_path_template = '.tools/templates/index.cn.html.json'
+        output_file_name = 'site.cn.json'
+
+    sections = []
+    sitemap = {"title": {lang: title}, 'sections': sections}
```
Comment (cs2be) [inline portal/deploy/sitemap_generator.py]: Should make each tutorial a section underneath the "Deep Learning 101" chapter

File: portal/deploy/sitemap_generator.py [hunk 3]
```diff
@@ -53,3 +55,47 @@ def models_sitemap(original_documentation_dir, generated_documentation_dir, vers
 
 def _get_destination_documentation_dir(version, output_dir_name):
     return '%s/docs/%s/%s' % (settings.EXTERNAL_TEMPLATE_DIR, version, output_dir_name)
+
+
+def _book_sitemap_with_lang(original_documentation_dir, generated_documentation_dir, version, output_dir_name, lang):
+    title = 'Deep Learning 101'
```
Comment (cs2be) [inline portal/deploy/sitemap_generator.py]: This should be Book, which is shown on top navigation

File: portal/deploy/sitemap_generator.py [hunk 4]
```diff
@@ -53,3 +55,47 @@ def models_sitemap(original_documentation_dir, generated_documentation_dir, vers
...
+    chapters = json_map['chapters']
+    for chapter in chapters:
+        parsed_section = {}
+
+        if chapter['name']:
+            parsed_section['title'] = {lang: chapter['name']}
+
+        if chapter['link']:
+            link = chapter['link']
+            link = link.replace('./', 'book/')
```
Comment (cs2be) [inline portal/deploy/sitemap_generator.py]: lets replace only at the start

**Pull request merged**